<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateNodeDialog</class>
 <widget class="QDialog" name="CreateNodeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string></string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="noteLabel">
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="text">
      <string></string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="aliasLabel">
     <property name="toolTip">
      <string>This is the name that will be used locally for your node</string>
     </property>
     <property name="text">
      <string>Node Alias</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QValidatedLineEdit" name="aliasEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="ipLabel">
     <property name="toolTip">
      <string>Please enter the IP address of your VPS in here, without a port</string>
     </property>
     <property name="text">
      <string>IP Address (without port)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QValidatedLineEdit" name="ipEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Label of transaction to yourself</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QValidatedLineEdit" name="labelEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="errorLabel">
     <property name="text">
      <string></string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QValidatedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qt/qvalidatedlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateNodeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateNodeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
